src/schema/resolvers/user-resolver.js

```diff
@@ -0,0 +1,12 @@
+import * as user from '../../controllers/user-controller.js'
+
+export default {
+  Query: {
+    users: (parent, args) => user.findUsers(),
+    user: (parent, { id }) => user.findUserById(id)
+  },
+  Mutation: {
+    createUser: (parent, args) => user.createUser(args),
+    deleteUser: (parent, id) => user.deleteUser(id)
+  }
```

Comment:
Please make the functions on the controller receive the arguments from Apollo directly. This file should only be a mapping to tell Apollo what function to execute for each field of the schema.
```javascript
// user-resolver.js
  Query: {
    users: user.findUsers,
    user: user.findUserById
  },
  Mutation: {
    createUser: user.createUser,
    deleteUser: user.deleteUser
  }

// models/user.js
export const findUsers = () => models.user.findAll()
export const findUserById = (_, { id }) => models.user.findByPk(id)
// ...etc
```